Server example (#17)
google/pybadges: server-example/README.md

## Simple Server

### Installing

Before running, it is **recommended** to install the versions of packages that were used when writing this code:

```sh
pip install -r requirements.txt
```

And if you don't have installed flask yet:

> ```sh

Reviewer: You shouldn't need to install flask seperately.

apt install python3-flask
```

### Running

> You must inform the environment variable FLASK_APP and then run:

Reviewer: ```suggestion
You must set the FLASK_APP environment variable before running the server:
```

```sh

> export FLASK_APP=hello.py

Reviewer: Isn't it called `app.py`?
